test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -704,7 +704,7 @@ def test_successful_money_transfer_payout_via_credit_individual_account_holder_t
 
   def test_successful_money_transfer_payout_via_credit_corporate_account_holder_type
     @credit_card.name = 'ACME, Inc.'
-    response = @gateway_oauth.credit(@amount, @credit_card, @payout_options.merge(account_holder_type: 'corporate'))
+    response = @gateway_oauth.credit(@amount, @credit_card, @payout_options.merge(account_holder_type: 'corporate', payout: true))
```

Comment:
Looking at the existing tests for payouts, I thought perhaps this should have been included in this test